### DISCUSSION TITLE
Новая система выбивания пушек копам

### DISCUSSION THREAD

**Discussion by FSBJI** [octothorp-team/public-dobrograd#648] [Предложения]
Теперь после попадания в руки копу, с него выпадает оружие которое у не было в руках в виде предмета. Но что бы этим не абузили, можно сделать что бы он был временым предметом и длился не больше 5-10минут

**Comment by Ggman77** [comment]
тогда краймеры будут постояно это абузить, я против

**Reply by FSBJI** [reply]
Где абуз? После выбивания пушку можно будет использовать до определенного времени после она пропадает.

**Comment by LENFYS** [comment]
Против

**Comment by travka4000** [comment]
как по мне такое себе, эсли бы реализовали как в платтсе я бы подумал

**Comment by SnakeLuck** [comment]
За. Так будет возможно будет выбивать пушку у полицейского и исключить такие приколы как /me взял пистолет с земли а дальше он устраивает шутфаер по челикам которые отвлеклись от него, будет возможно откинуть пистолет так чтобы полисмен не смог сделать так как я описал
